client/client_test.go

```diff
@@ -144,6 +155,15 @@ func (s *ClientSuite) TestInitRootTooLarge(c *C) {
 	c.Assert(client.Init(s.rootKeys(c), 0), Equals, ErrMetaTooLarge{"root.json", maxMetaSize + 1})
 }
 
+func (s *ClientSuite) TestInitRootExpired(c *C) {
+	duration := 100 * time.Millisecond
```

Comment:
I had TravisCI in the back of my mind, knowing how slow it can be sometimes. Happy to reduce it, tests only take ~1.3s with this delay though